doc/compodoc_sources/concepts/configuration.md

```diff
@@ -343,6 +343,32 @@ The inner object can have the following but not always required fields: `"input"
   }
 ```
 
+#### The PreviousSchools Component
```

Comment:
We should find a different place to describe all the custom sections soon, otherwise this page will become unusably long